package.json

```
@@ -42,6 +42,7 @@
     "logops": "2.1.0",
     "mqtt": "3.0.0",
     "request": "2.88.0",
+    "sinon": "^9.0.1",
```

```suggestion
    "sinon": "~9.0.1",
```

Why this new depedency is needed? If related with testing it should be moved to `devDependencies`. I understand that sinon is something related with utest tooling... Cannot the current ones (mocha and nock) cover the same functionality?

---

Fixed 240d5f9 Moved to `devDependencies` - it is required for mocking the file system.  Nock is used for HTTP mocks. Sinon was suggested via the previously commented code in the relevant test in the JSON Agent.

:bar_chart: Got to keep that code coverage up. :chart_with_upwards_trend: 84% and rising